packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -147,6 +147,13 @@ class _TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
     _entry = new OverlayEntry(builder: (BuildContext context) => overlay);
     Overlay.of(context, debugRequiredFor: widget).insert(_entry);
     GestureBinding.instance.pointerRouter.addGlobalRoute(_handlePointerEvent);
+    switch (defaultTargetPlatform) {
```

Comment:
I'd probably always send this event and leave it up to the platform side to either drop it on the floor or announce it.